src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -778,11 +778,11 @@ public static boolean contentEqualsIgnoreEOL(final Reader input1, final Reader i
 
         String line1 = br1.readLine();
         String line2 = br2.readLine();
-        while (line1 != null && line2 != null && line1.equals(line2)) {
+        while (line1 != null && line1.equals(line2)) {
             line1 = br1.readLine();
             line2 = br2.readLine();
         }
-        return line1 == null ? line2 == null ? true : false : line1.equals(line2);
+        return line1 == null && line2 == null;
```

Comment:
What about the `line1.equals(line2)` bit?

Reply:
yes,  you way is better than me